spec/lib/account_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe BankAccounts::Account do
```

Comment:
If you want to avoid having to type out `BankAccounts::Account` everywhere, you can wrap the whole file in `module BankAccount`, the same way you did your classes, and everything should be namespaced properly.